can-bus/json/DCM/DCM_tx.json

```diff
@@ -35,28 +34,23 @@
                 "offset": 0,
                 "min": -32768,
                 "max": 32767,
-                "unit": "rpm",
-                "start_value": 0
+                "unit": "rpm"
             },
             "DirectionCommand": {
                 "start_bit": 32,
-                "enum": "InverterDirection",
-                "start_value": 0
+                "enum": "InverterDirection"
             },
```

Comment:
I don't think you meant to commit this? This is the outdated version of the DCM_tx.json file, pls revert to whatever is in f4-port

Reply:
(Except for the stack watermark check for 100Hz)

Reply:
Oh shoot good catch, yeah this def wasn't intended